src/components/Card/Card.tsx

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react';
 import {useNavigate} from 'react-router-dom';
 import {ListItemColumnType, TeamsType, UserDataType} from 'types/types';
-import {CardContainer} from './Card.styled';
+import {CardBody, CardBodyContent, CardContainer, CardAvatar, CardLeader, CardText, CardTitle} from './Card.styled';
 
 interface CardProps {
     id: string;
     url?: string;
     columns: ListItemColumnType[];
     hasNavigation?: boolean;
     navigationProps?: UserDataType | TeamsType;
+    isUser?: boolean;
+    isLeader?: boolean;
```

Comment:
Needed to create this props to differentiate the card for it's possible types, which are Team, User, and User Team Leader